src/InstrumentationEngine.NuGet/InstrumentationEngine.Nuget.csproj

```diff
@@ -27,7 +27,7 @@
     <RootNamespace>Microsoft.Diagnostics.InstrumentationEngine.NuGet</RootNamespace>
     <!-- NU5100: Package is used as a means to acquire the engine and not to have its assets referenced directly. -->
     <NoWarn>$(NoWarn);NU5100</NoWarn>
-    <Platforms>AnyCPU;x64;x86</Platforms>
+    <Platforms>AnyCPU;x64;x86;arm64</Platforms>
```

Comment:
I think this should be `ARM64` to be consistent with the idea that Platform is set as upper case by is lowercase when used in file system naming.